test/fixtures/addressesUsingIdsQuery/no_layers_with_boosts.json

```diff
@@ -0,0 +1,63 @@
+{
+  "query": {
+    "function_score": {
+      "query": {
+        "bool": {
+          "minimum_number_should_match": 1,
+          "should": [
+            {
+              "bool": {
+                "_name": "fallback.address",
+                "boost": 17,
+                "must": [
+                  {
+                    "match_phrase": {
+                      "address_parts.number": "housenumber value"
+                    }
+                  },
+                  {
+                    "match_phrase": {
+                      "address_parts.street": "street value"
+                    }
+                  }
+                ],
+                "filter": {
+                  "term": {
+                    "layer": "address"
+                  }
+                }
+              }
+            },
+            {
+              "bool": {
+                "_name": "fallback.street",
+                "boost": 19,
+                "must": [
+                  {
+                    "match_phrase": {
+                      "address_parts.street": "street value"
+                    }
+                  }
+                ],
+                "filter": {
+                  "term": {
+                    "layer": "street"
+                  }
+                }
+              }
+            }
+          ],
+          "filter": {
+            "bool": {
+              "minimum_number_should_match": 1,
+              "must": []
+            }
+          }
+        }
+      },
+      "functions": []
+    }
+  },
+  "size": { "$": "size value" },
+  "track_scores": { "$": "track_scores value" }
```

Comment:
I don't think the `$` is correct here? that looks like an artifact of the `Variable` object, probably because it's not being serialized in the intended way.